plugins/search/src/components/SearchType/SearchType.Accordion.test.tsx

```diff
@@ -15,29 +15,30 @@
  */
 
 import React from 'react';
-import { ApiProvider } from '@backstage/core-app-api';
-import { TestApiRegistry } from '@backstage/test-utils';
+import { TestApiProvider } from '@backstage/test-utils';
 import { act, render } from '@testing-library/react';
 import user from '@testing-library/user-event';
-
-import { searchApiRef } from '../../apis';
-import { SearchContext, SearchContextProvider } from '../SearchContext';
+import {
+  searchApiRef,
+  SearchContextProvider,
+} from '@backstage/plugin-search-react';
 import { SearchType } from './SearchType';
 
-describe('SearchType.Accordion', () => {
-  const query = jest.fn();
-  const mockApis = TestApiRegistry.from([searchApiRef, { query }]);
+const setTypesMock = jest.fn();
+const setPageCursorMock = jest.fn();
 
-  const contextSpy = {
-    result: { loading: false, value: { results: [] } },
-    term: '',
+jest.mock('@backstage/plugin-search-react', () => ({
```

Comment:
hmm I'm not sure I understand, by using the test provider we mock the search api, by mocking `useSearch` we get access to state and state setters available on the context so that we can use that in the assertions.

Reply:
Aaaah, I understand now.  Can't spy on the setters in the context because the context is now private to `@backstage/plugin-search-react`.

Yeah, maybe mocking the hook is the simplest way to do this.  Otherwise you'd have to set up a little test component that uses the real `useSearch` hook and renders some of its values...  And then modify the assertions to look at the values instead of spy on the setters.  🤷‍♂️ 

This seems fine for now!